src/server/createDescriptor.ts

```diff
@@ -5,7 +5,7 @@ import { loadSync, Options } from '@grpc/proto-loader';
 
 export default function createDescriptor(protoFile: string): GrpcObject {
   const loaderOptions: Options = {
-    keepCase: true,
+    keepCase: false,
```

Comment:
@dushanpantic can this be changed from outside via the command?

Reply:
I will make this whole object customizable from outside

Reply:
btw, where would you find this change useful?

Reply:
When you have different property naming when converting eg. from Porto files to TS

Reply:
I have added loading of custom loader options in the latest version. 
see: https://github.com/dushanpantic/grpc-server-mock#custom-loader-options
and example here: https://github.com/dushanpantic/grpc-server-mock/tree/master/fixtures/autowire/custom-loader-options

Reply:
Thank you very much - that's awesome. I will try it out